sourcemash/frontend/templates/item-card.tpl

```diff
@@ -15,8 +15,11 @@
       </div>
 
       <div class="card-action valign-wrapper row">
-        <div class="col s4"><a href="/#categories/<%=item.get('category_1')%>"><%=item.get('category_1')%></a></div>
-        <div class="col s4"><a href="/#categories/<%=item.get('category_2')%>"><%=item.get('category_2')%></a></div>
+        <div class="category-list flow-text col s8">
```

Comment:
flow-text only works as a class on a <p>. It's what makes the text responsive (see browse feed page)